atlp-rwanda/elites-bn-be
# Ft #181339632 mark all notifications

# What does this PR do?
Mark as read all notifications.

# Description of Task to be completed?
- [x]  Mark as read all notifications.
- [x] swagger documentation

# How should this be manually tested?
1.  Clone the repository, do checkout to the branch ft-#181339632-mark-all-notifications, configuration needed dependencies by following instructions from readme.md file.
2. To run first run the "yarn dev", into the browser url 
http://localhost:xxx/docs/swagger-ui
xxx as port where our server is running.
3. Login for the registered user who has notification to mark.
4.  check the markallasread endpoint.

## How Has This Been Tested?


Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Should not mark all as read if he has not logged in
- [x] Should mark all as read

# Any background context you want to provide?
N/A

# What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/181339632

# Screenshots (if appropriate)
![Screenshot from 2022-04-08 10-30-21](https://user-images.githubusercontent.com/44385819/162410625-57660c0e-da86-4780-849f-1413cf98bf59.png)
![Screenshot from 2022-04-08 11-12-11](https://user-images.githubusercontent.com/44385819/162410631-37c310f7-9b9c-4664-9de1-6c2dc46f07a4.png)
![Screenshot from 2022-04-08 11-14-53](https://user-images.githubusercontent.com/44385819/162410635-47a234bd-b06c-4af4-a0d5-6f040fc2fcea.png)
![Screenshot from 2022-04-08 11-21-59](https://user-images.githubusercontent.com/44385819/162410642-cc24efd5-3d6b-4c6d-a752-596eeeaa8c44.png)
![Screenshot from 2022-04-08 11-27-19](https://user-images.githubusercontent.com/44385819/162410647-43a86e84-f701-4ccb-bba8-fb876abd22f9.png)

# Questions:
N/A
